### PR TITLE
Speed Up First Meaningful Paint

### PR DESCRIPTION
**Pull request by deadlybyte**
Speed improvements to website, include compressing/optimise images that weren't being optimised. Also added font-display: swap property to the CSS.

### CODE REVIEW

File: src/components/project-list.jsx [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
-    <img src={logo} alt="Howly logo" />
-    <h2>
-      React Native, Node.js, AWS,
-      <br /> Serverless, Product Design
-    </h2>
-    <p>The snapchat of events, Howly keeps you and your friends connected</p>
-  </ProjectContainer>
-)
+const Project = () => {
+  const data = useStaticQuery(graphql`
```
Comment (mdcuk34) [inline src/components/project-list.jsx:10]: Not sure if this should be within the component function or not?

File: src/components/project-list.jsx [hunk 2]
```diff
@@ -1,21 +1,43 @@
...
+    query {
+      howlyLogoImg: file(relativePath: { eq: "howly-logo.png" }) {
+        childImageSharp {
+          fluid(maxHeight: 1000) {
+            ...GatsbyImageSharpFluid_tracedSVG
```
Comment (mdcuk34) [inline src/components/project-list.jsx:15]: I’ve been using blurred vs traced for the rest of the site

File: src/pages/howly.jsx [hunk 3]
```diff
@@ -41,13 +44,21 @@ export const pageQuery = graphql`
...
-        fluid(maxHeight: 1000) {
+        fluid(maxHeight: 1000, quality: 50) {
           ...GatsbyImageSharpFluid
         }
       }
     }
+    howlyLogoImg: file(relativePath: { eq: "howly-logo.png" }) {
+      childImageSharp {
+        fluid(maxHeight: 1000) {
+          ...GatsbyImageSharpFluid_tracedSVG
```
Comment (mdcuk34) [inline src/pages/howly.jsx:55]: Blurred

**Review by mdcuk34 — APPROVED** [review]

File: src/components/project-list.jsx (hunk 2 above)
Comment (deadlybyte) [reply src/components/project-list.jsx:15]: No worries, I'll change it, thought it looked better using traced though! As it outlined the word Howly.

File: src/pages/howly.jsx (hunk 3 above)
Comment (deadlybyte) [reply src/pages/howly.jsx:55]: No worries, I'll change it, thought it looked better using traced though! As it outlined the word Howly.

File: src/components/project-list.jsx (hunk 1 above)
Comment (deadlybyte) [reply src/components/project-list.jsx:10]: Yeah, I had it outside, but most of the examples were using the useStaticQuery, so went with that approach, Can change if you think having it outside is better.

Comment (mdcuk34) [reply src/components/project-list.jsx:10]: Roger that, I’m easy... I guess whatever is more performant although would be nice to use the same pattern throughout.

**Pull request merged**